code/game/objects/items/tools/misc_tools.dm

```diff
@@ -12,43 +12,30 @@
 	icon_state = "labeler0"
 	item_state = "flight"
 	var/label = null
-	var/labels_left = 50
 	var/mode = 0	//off or on.
-
+	var/labels_left = 50
```

Comment:
You still have changes here and honestly removing the space before new definitions just makes things harder to read